src/components/screens/TimeTables.tsx

```diff
@@ -28,15 +29,15 @@ export const TimeTables = ({
   iconName: keyof typeof Icon;
 }) => {
   const CategoryIcon = Icon[_upperFirst(iconName)];
-  const today: string | number = new Date().toISOString();
+  const today: string = moment().format('ddd, DD.MM.YYY');
 
   return (
     <>
       <SectionHeader title={texts.pointOfInterest.departureTimes} />
 
       <Wrapper style={styles.noPadding}>
         <BoldText small>
-          {texts.pointOfInterest.today} {momentFormat(today, 'ddd, DD.MM.YYYY')}
+          {texts.pointOfInterest.today} {today}
```

Comment:
so much better to read in both places, `moment` on top and nothing but `today` down here.